### ISSUE TITLE
Error running "npm install" trying to setup reactide

### ISSUE THREAD

**Issue by timk519** [reactide/reactide#219]
I was going through the install instructions and got the following error on "npm install"

# npm resolution error report

2022-08-02T21:47:27.070Z

While resolving: reactide@3.0.0
Found: react@16.14.0
node_modules/react
  react@"^16.6.0" from the root project
  peer react@"^0.14.0 || ^15.0.0-0 || ^16.0.0-0" from chai-enzyme@1.0.0-beta.1
  node_modules/chai-enzyme
    chai-enzyme@"^1.0.0-beta.1" from the root project

Could not resolve dependency:
peer react@"0.13.x || 0.14.x || ^15.0.0-0 || 15.x" from enzyme@2.9.1
node_modules/enzyme
  enzyme@"^2.8.0" from the root project
  peer enzyme@"^2.7.0 || ^3.0.0" from chai-enzyme@1.0.0-beta.1
  node_modules/chai-enzyme
    chai-enzyme@"^1.0.0-beta.1" from the root project

**Comment by Arian1192** [comment]
same problem

**Comment by ChristopherLeitner** [comment]
Yeah, having the same issue.

**Comment by TechnoFynch** [comment]
yep, same issue for me as well.

**Comment by nilslindkvist** [comment]
I also have this issue, guess I'm going over to VSCode for now.

**Comment by gitRacoon** [comment]
the same issue. i used --legacy-peer-deps to fix

**Comment by preitinger** [comment]
Same issue. Is this project dead? And if it is dead, and React recommends this IDE. Is then React dead, too?

**Comment by TechnoFynch** [comment]
> Same issue. Is this project dead? And if it is dead, and React recommends this IDE. Is then React dead, too?

Nope... This project may be dead but there are several other IDEs that support React. Also, React has a very large growing community, so I doubt it will die in the near future.

**Comment by TechnoFynch** [comment]
> the same issue. i used --legacy-peer-deps to fix

But then the build is giving errors. I tried this too.

**Comment by gitRacoon** [comment]
> But then the build is giving errors. I tried this too.

I got this error after install with --legacy-peer-deps: https://github.com/reactide/reactide/issues/218

**Comment by preitinger** [comment]
I think it requires react 15 but normally react 16 is installed nowadays. can this be the problem?

**Comment by preitinger** [comment]
OK I will code with atom...
